docs/datasync/soft-delete.md

```diff
@@ -8,7 +8,7 @@ This is the simplest strategy for Data Synchronization that is also quite straig
 
 ## Setup
```

Comment:
I will rename soft deletes (this file) to just become delta thing as there is no other strategy we do for the moment. (it is always soft deletes that are used for delta.